specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -6040,6 +6044,10 @@
           "$ref": "#/definitions/NetworkPlugin",
           "description": "networkPlugin for managed cluster snapshot."
         },
+        "networkPluginMode": {
+          "$ref": "#/definitions/NetworkPluginMode",
+          "description": "networkPluginMode for managed cluster snapshot."
```

Comment:
`NetworkPluginMode` (cap first letter)

Reply:
any reason? shouldn't it follow the other properties in networkProfile?

Reply:
I mean in the doc not the API. And... I suppose yes it should but the other fields in `NetworkProfileForSnapshot` don't follow the pattern of the broader Swagger I don't think? (I thought most everything was capitalized first?)

Reply:
> I mean in the doc

which doc are you referring? I did notice that the enum "name" was not capitalized so here's a pr for fix that: https://github.com/Azure/azure-rest-api-specs/pull/18884

Reply:
> networkPluginMode for managed cluster snapshot.

to

> NetworkPluginMode for managed cluster snapshot.

Was all I was suggesting. You're right that the other descriptions here in this particular type don't cap the first letter but I think they're all wrong actually. It's really minor though.

Reply:
makes sense, I added that change in the open PR linked above just now.